package io.kyberorg.yalsee.exception;

import io.kyberorg.yalsee.ui.err.ServerErrorView;

/**
 * Very general exception, which not intended to be thrown, just {@link ServerErrorView} ui handles it.
 *
 * @since 2.7
 */
public class GeneralServerException extends YalseeException {
}
